TwitterClone/sub_twitter/templates/sub_twitter/home.html

```diff
@@ -1,129 +1,144 @@
-{% load static %}
-{% load cloudinary %}
 <!DOCTYPE html>
+{% load static %} {% load cloudinary %}
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="icon" href="{% static '/img/droplet.svg' %}">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="{% static 'img/droplet.svg' %}" />
     <!-- bootstrap/css -->
-    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
+    <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}" />
 
     <!--FontAwesome -->
-    <script src="https://kit.fontawesome.com/da8acf5d2d.js" crossorigin="anonymous"></script>
-
+    <script
+      src="https://kit.fontawesome.com/da8acf5d2d.js"
+      crossorigin="anonymous"
+    ></script>
 
     <title>Reante-Twitter</title>
+  </head>
+
+  <body class="home" style="background-color: rgba(26, 91, 121, 0.5)">
+    <nav class="navbar navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">
+        <img
+          src="{% static 'img/favicon.png' %}"
+          width="40"
+          height="40"
+          class="d-inline-block align-mid"
+          alt=""
+        />
+        Home
+      </a>
+    </nav>
 
-</head>
-
-<body class="home" style="background-color:rgba(26, 91, 121, .5);">
-  <nav class="navbar navbar-dark bg-dark" >
-    <a class="navbar-brand" href="#">
-      <img src="{% static '\img\favicon.png' %}" width="40" height="40" class="d-inline-block align-mid" alt="">
```

Comment:
`\img\favicon.png` is an invalid path to your file! Always use forward slashes(/) when referring to a file as that's the UNIX way, which is what your heroku server will run on. Also, do not add the first / before image since it should NOT refer to the root directory, instead it should be from the static folder. so the correct one would be `{% static 'img/favicon.png' %}`

Notice that there is no / before img. This is one of the common mistakes that I found in this file! You might want to check and correct this in other 2 template files as well!